compoments/report-repair/contact-details.js

```diff
@@ -20,18 +20,31 @@ const ContactDetails = ({handleChange, values}) => {
   }
 
   const options =  [
-    { value: 'text', title: 'Text message (recommended)', conditional: {
-      label: 'Please enter a UK mobile number',
-      type: 'tel', validator: mobilePhoneNumberValidator, onKeyPress: phoneOnKeyPress,
-      emptyInputErrorMessage: 'Enter a UK mobile number',
-      invalidInputErrorMessage: 'Enter a valid UK mobile number',
-    }},
-    { value: 'email', title: 'Email', conditional: {
-      label: 'Please enter your email address',
-      type: 'email', validator: emailValidator,
-      emptyInputErrorMessage: 'Enter an email address',
-      invalidInputErrorMessage: 'Enter a valid email address',
-    }}
+    {
+      value: 'text',
+      title: 'Text message (recommended)',
+      conditional: {
+        autoComplete: 'tel-national',
```

Comment:
@Bhavesh-Patel I've found this example https://design-system.service.gov.uk/patterns/telephone-numbers/ just now and it's using `autocomplete="tel"`. Do you think I should replace to match the design systems?

Reply:
All done!